src/Analysis/GWASV2/Components/Diagrams/CohortsOverlapDiagram/CohortsOverlapDiagram.jsx

```diff
@@ -8,15 +8,17 @@ import {
   addCDFilter,
 } from '../../../Shared/cohortMiddlewareApi';
 import Simple3SetsEulerDiagram from './Simple3SetsEulerDiagram';
+import { useSourceContext } from '../../../Shared/Source';
 
 const CohortsOverlapDiagram = ({
-  sourceId,
   selectedStudyPopulationCohort,
   selectedCaseCohort,
   selectedControlCohort,
   selectedCovariates,
-  selectedDichotomousCovariates,
+  outcome,
 }) => {
+  const { source } = useSourceContext();
+  const sourceId = source; // TODO - change name of source to sourceId for clarity
```

Comment:
I created a ticket for this new issue here: https://ctds-planx.atlassian.net/jira/software/projects/VADC/boards/132/backlog?selectedIssue=VADC-360